spec/enumerable_spec.rb

```diff
@@ -0,0 +1,106 @@
+# rubocop: disable Layout/LineLength
```

Comment:
- [OPTIONAL] Disabling cops such as `Layout/LineLength` should be avoided as they could be easily fixed. 
    For example:
```ruby
expect(words.my_each_with_index { |item, index| puts "#{item} : #{index}" }).to eql(words.each_with_index { |item, index| puts "#{item} : #{index}" })
```
could be divided into two lines like this(note that we use indentation to specify continuation):
```ruby
expect(words.my_each_with_index { |item, index| puts "#{item} : #{index}" })
        .to eql(words.each_with_index { |item, index| puts "#{item} : #{index}" })
```

Reply:
Solved!